lib/util/cpu-profiler.js

```diff
@@ -0,0 +1,35 @@
+const inspector = require('inspector');
+const fs = require('fs');
+
+const DEFAULT_DURATION = 30000; // 30 sec
```

Comment:
maybe 30_SECONDS would be better name? (instead of comment)

Reply:
the name of variable is fine, unnecessery comment, so to delete ;)